src/views/CardStack.js

```diff
@@ -343,7 +331,15 @@ class CardStack extends Component<DefaultProps, Props, void> {
 
     let panHandlers = null;
 
-    if (this.props.gesturesEnabled) {
+    const cardStackConfig = this.props.router.getScreenConfig(
+      props.navigation,
+      'cardStack'
+    ) || {};
+    const gesturesEnabledConfig = cardStackConfig.gesturesEnabled;
+    const gesturesEnabled = typeof gesturesEnabledConfig === 'boolean' ?
+      gesturesEnabledConfig :
+      Platform.OS === 'ios';
```

Comment:
This logic looks good to me, but can you write a comment describing the platform defined default?